pages/components/Lists/lists.css

```diff
@@ -0,0 +1,46 @@
+.list {
+    border: 1px solid #e6e6e9;
+    padding-left: 0;
+    padding: 1em;
+    border-radius: 0.5em;
+    width: 17em;
```

Comment:
Hey, if possible, avoid fixed widths... You can let the user define by himself. However, for demonstration purpose, it's fine.
Make it as un-opinionated as possible